packages/sdk/src/adapters/multipeer/session.ts

```diff
@@ -257,6 +257,16 @@ export class Session extends EventEmitter {
             // Merge the update into the existing actor.
             syncActor.initialization.message.payload.actor
                 = deepmerge(syncActor.initialization.message.payload.actor, message.payload.actor);
+
+            // strip out transform data that wasn't updated
+            // so it doesn't desync from the updated one
+            const cacheTransform = syncActor.initialization.message.payload.actor.transform;

```

Comment:
>cacheTransform [](start = 18, length = 14)

I believe it's possible for this to be undefined, depending on how the syncActor was created.

Reply:
It will only be undefined if patchTransform is also undefined, because it's after the deepmerge.

---
In reply to: [299102576](https://github.com/microsoft/mixed-reality-extension-sdk/pull/361#discussion_r299102576) [](ancestors = 299102576)

Reply:
That's true. Thanks.

---
In reply to: [299124936](https://github.com/microsoft/mixed-reality-extension-sdk/pull/361#discussion_r299124936) [](ancestors = 299124936,299102576)